packages/scrapers-v2/src/main.ts

```diff
@@ -0,0 +1,21 @@
+import { scrapeMajorLinks } from "./urls/urls";
+import { classifyCatalogEntries } from "./classify/classify";
+import { CatalogEntryType } from "./classify/types";
+import { filterByEntryType, flattenCatalogHierarchy } from "./utils";
+import { fetchAndTokenizeHTML } from "./tokenize/tokenize";
+
+export const runScrape = async () => {
+  try {
+    const hierarchy = await scrapeMajorLinks(2021, 2022);
+    const flattenedUrls = flattenCatalogHierarchy(hierarchy);
```

Comment:
Is there a reason why we build up the hierarchy object in `scrapeMajorLinks` and then flatten again? Why not just take the last "parts" from the `scrapeMajorLinks` before building an hierarchy?

Reply:
it's because in the future, it might be helpful to actually keep the hierarchy information. right now we don't need it so we just flatten it